app/controllers/accounts_controller.rb

```diff
@@ -68,20 +68,27 @@ def get_profile
     if @current_user
       user = User.find_by(id: params[:id].to_i)
 
-      if not user
+      unless user
         render(inertia: 'NotFound')
         return
       end
 
+      user_json = user.as_json(only: [:id, :name, :username, :description, :avatar])
```

Comment:
this conversion should be done as late as possible

Reply:
why? In case there is an exception?

Reply:
it's just bad practise. format conversions should be right before they are used or sent back. otherwise it's harder to read and also more error prone as we might change data in the meantime